### PR TITLE
fix: support asdf manager, repair db setup

### PR DESCRIPTION
**Pull request by Aceba1**
- Adds support for asdf runtime management through `.tool-versions` file
- Fixes database setup failure from missing `location` definition

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ruby 2.5.7
+yarn 1.22.4
+nodejs 14.3.0
```
Comment (geraldalewis) [inline .tool-versions:3]: `git` is happiest when files end in a newline -- add one here?

File: app/models/traffic_cam.rb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 class TrafficCam < ApplicationRecord
+  belongs_to :location
```
Comment (geraldalewis) [inline app/models/traffic_cam.rb:2]: I could see this being its own PR, or updating the title/description of this PR to reflect that you're doing two different things.

**Review by geraldalewis — COMMENTED** [review]
Beautiful PR! Two style-guide type comments, but otherwise LGTM 👍

**Review by cbarber — APPROVED** [review]
lgtm

**Comment by Aceba1** [comment]
🎉

**Pull request merged**